tests/PKSim.Tests/Core/IndividualScalingTaskSpecs.cs

```diff
@@ -122,9 +133,9 @@ public void should_leverage_the_scaling_method_task_to_retrieve_the_default_scal
       }
 
       [Observation]
-      public void should_not_return_the_reference_concentration_parameter()
```

Comment:
Interesting that we were EXPLICITELY excluding ref concentration from scaling...not sure why. Probably because someone said it did not make sense? Well now other someones want to have it :)